examples/i18n/src/resource.rs

```diff
@@ -0,0 +1,66 @@
+use std::collections::HashMap;
+
+mod german;
+
+#[derive(Debug)]
+pub struct Resource(HashMap<String, String>);
+
+impl Default for Resource {
+    fn default() -> Self {
+        let ftl = String::from(
```

Comment:
I assume the FTL below are English translations => extract it to the file `english.ftl` (see review comment about `german.ftl` below)

Reply:
Yes! Done.